### ISSUE TITLE
Incrementally Adjust DSK Mask parameters in Blackmagic ATEM

### ISSUE THREAD

**Issue by bryancj** [bitfocus/companion#1821]
**Describe the feature**

I'm would like a way to Incrementally Adjust DSK Mask in blackmagic ATEM switchers.

What I mean by this is I want a function that can adjust a given mask parameter by +/- a given amount . In other words, The current mask for the top is set to -5.0 and executing this command would add 0.5 to that increasing it to -4.5. Executing it a second time would then increase it to -4.0 and so on. 

I have played around with doing this in a macro but I can only find a way to set the mask parameters in absolute positions, not relative. 

I know this can be done by going to the settings in the screen on the actual ATEM switcher and adjusting the top mask number with the rotary encoder or through the ATEM software control, however, I am looking for a way to do this through a Bitfocus Companion instance. 

Thank you in advanced. 

**Usecases**
Could be used to easily adjust mask parameters on the fly for less technical operators.

**Comment by estilles** [comment]
@bryancj 
Are you using the [bmd-atem](https://github.com/bitfocus/companion-module-bmd-atem) module?

**Comment by bryancj** [comment]
@estilles 
Thank you for checking. Yes, I'm using the ATEM Module in Companion, Just haven't been able to find a way to adjust the DSK Mask without creating a Macro in the ATEM software control. Even using macros, I can only find a way to set absolute values. If it helps any, I'm using an ATEM Television Studio Pro 4K.

**Comment by estilles** [comment]
@bryancj 
You're right. There is currently now a way to do it at this time. I'm goin to transfer this issue to the corresponding repo.
